containers/jekyll/README.md

```diff
@@ -6,20 +6,46 @@
 
 | Metadata | Value |  
 |----------|-------|
-| *Contributors* | [Carlos Mendible](https://github.com/cmendible) |
+| *Contributors* | [Carlos Mendible](https://github.com/cmendible), [parkr](https://github.com/parkr) |
 | *Categories* | Community, Languages, Frameworks |
 | *Definition type* | Dockerfile |
+| *Published images* | mcr.microsoft.com/vscode/devcontainers/jekyll |
+| *Published image architecture(s)* | x86-64 |
 | *Works in Codespaces* | Yes |
 | *Container host OS support* | Linux, macOS, Windows |
 | *Container OS* | Debian |
 | *Languages, platforms* | Ruby, Jekyll |
 
+See **[history](history)** for information on the contents of published images.
+
+## Using this definition
+
 In addition to Ruby and Bundler, this development container installs Jekyll and the required tools at startup:
 
 - If your Jekyll project contains a `Gemfile` in the root folder, the development container will install all gems at startup by running `bundle install`. This is the [recommended](https://jekyllrb.com/docs/step-by-step/10-deployment/#gemfile) approach as it allows you to specify the exact Jekyll version your project requires and list all additional Jekyll plugins.
 - If there's no `Gemfile`, the development container will install Jekyll automatically, picking the latest version. You might need to manually install the other dependencies your project relies on, including all relevant Jekyll plugins.
 
-## Using this definition
+You can decide how often you want updates by referencing a [semantic version](https://semver.org/) of each image. For example:
+
+- `mcr.microsoft.com/vscode/devcontainers/jekyll:0`
+- `mcr.microsoft.com/vscode/devcontainers/jekyll:0.0`
+- `mcr.microsoft.com/vscode/devcontainers/jekyll:0.0.1`
```

Comment:
How do you intend to version the Jekyll container? Since it's not presently tied to a Jekyll or GitHub Pages version, and because the post-install script will still run `bundle install`, it may not need to be versioned at all.

Reply:
Yeah this versioning is for the image contents. But if we don't have a reason to release, we wouldn't bump the version number up.

At a minimum, there's monthly OS patching, so we'd update roughly once a month and that would get the latest of everything. That just increments the last digit (since we use semver). We do the same thing for all images in the repo. [This is explained here](https://github.com/microsoft/vscode-dev-containers/issues/532).

The other thing this does is future proof for when/if we need to make breaking changes.